crates/mirror-mirror/src/value.rs

```diff
@@ -57,6 +58,7 @@ pub enum Value {
     TupleStructValue(TupleStructValue),
     TupleValue(TupleValue),
     List(Vec<Value>),
+    Set(UnorderedSet<Value>),
```

Comment:
Not totally sure which set type we should use here 🤔 ordered/unordered

Reply:
I would say Ordered I think. Maybe even a LinearSet. Large sets probably shouldn't be getting reflected anyway and can just skip that field, no?

Reply:
Hmm, that sounds like an oversight. LinearSet should trivially be able to impl Hash

Reply:
Ah okay. I'll make a PR for that then :)